### PR TITLE
Setup message

### PR DESCRIPTION
**Pull request by alexgraham-da**
Adds a "setup required" screen if the Operator trigger is not deployed and the Operator role contract is not created. Also removes some of the flashing screens when logging in.

### CODE REVIEW

File: ui/src/components/App.tsx [hunk 1]
```diff
@@ -70,20 +70,5 @@ const App: React.FC = () => {
   )
 }
 // APP_END
-
-const PublicProvider: React.FC = ({ children }) => {
```
Comment (alexgraham-da) [inline ui/src/components/App.tsx:74]: This wasn't being used anymore since @alexmatson-da changed to the custom query stream provider

**Review by Sarahb-da — APPROVED** [review]
lgtm!

File: ui/src/components/LoginScreen.tsx [hunk 2]
```diff
@@ -57,11 +83,25 @@ const LoginScreen: React.FC<Props> = ({onLogin}) => {
...
 
-  const tiles = deploymentMode === DeploymentMode.PROD_DABL ? dablTiles : localTiles;
+  const setupRequiredTile = (
+    <Tile key='test' header={logoHeader}>
+      <div className='setup-required'>
+        <h3>Welcome to the Daml Open Marketplace!</h3>
+        <p>
+          It looks like you have not completed the necessary deployment steps to configure this app.
+          Please create an Operator role contract for the UserAdmin party, and deploy the necessary triggers.</p>
+        <h4>See <a href='https://github.com/digital-asset/da-marketplace#add-the-operator-role-contract'>here</a> for more information.</h4>
```
Comment (alexmatson-da) [inline ui/src/components/LoginScreen.tsx:93]: btw in the screenshot from the other day, the default blue/purple link color looked pretty hard to read against the dark background.

Maybe add the className `"dark"` to the `a`, and have the color of `a.dark` tags be [var(--blue-80)](https://www.figma.com/file/HxPETLG8mqxF00IIod5gY6/Daml-Open-Marketplace-WF?node-id=1080%3A1482) somewhere in css

**Review by alexmatson-da — APPROVED** [review]

File: ui/src/components/LoginScreen.tsx (hunk 2 above)
Comment (alexgraham-da) [reply ui/src/components/LoginScreen.tsx:93]: that looks much better!

**Pull request merged**